app/assets/javascripts/float-kit/addon/components/d-toast.gjs

```diff
@@ -61,18 +61,29 @@ export default class DToast extends Component {
     }).finished;
   }
 
+  get duration() {
+    let duration = this.args.toast.options.duration;
+
+    if (duration === "long") {
+      return 5000;
+    } else {
+      return 3000;
+    }
+  }
```

Comment:
Some plugins use toasts, so maybe we need a backwards compatible code path with a deprecation warning?

Reply:
Yes, I hesitated, but I think in this case, it's not breaking the application, it's mostly a UX detail. Maybe I could get convinced that 5000 is a better default?

Reply:
You know what? OK let's show a deprecation message, I don't really mind and we can just remove it in 6 months.